src/content-handlers/iiif/extensions/uv-openseadragon-extension/DownloadDialogue.tsx

```diff
@@ -561,7 +602,7 @@ const DownloadDialogue = ({
           <ol className="options">
             {isDownloadOptionAvailable(DownloadOption.CURRENT_VIEW) && (
               <li className="option single">
-                <button
+                <button ref={firstFocusableElementRef}
```

Comment:
Because of the isDownloadOptionAvailable check around each button, there is no guarantee about which buttons will actually be rendered -- thus, I think we're going to need somewhat more sophisticated logic to figure out which will be the first and last focusable elements. Maybe we need to refactor this to build an array of buttons and then build the ol/li from the array, so we can apply the references in the appropriate places.